### ISSUE TITLE
Some Feedback

### ISSUE THREAD

**Issue by araffin** [AtsushiSakai/PythonRoboticsPaper#1]
Hi,
here are some feedback about the paper.

Note: all the links (references) are not clickable, is that normal ?

I wrote my comments in the paper order:

## Abstract and Introduction

Open Source Software(OSS) -> Open Source Software (OSS)
It aims for beginners of robotics -> It aims beginners of robotics
(It is made/meant for)
In this project, the algorithms which are practical and widely used in both academia and industry are selected
-> the proposed algorithms are practical and ...
is written by Python3 -> is written in Python3
only depends some -> only depends on some 
for readability and easy to use -> ... and ease of use.

"In this paper, related works of this project,
some key ideas about this OSS project, and the brief structure of this repository are introduced. Future works of this project are also discussed."
Not sure if it should be in the abstract, sound more like introduction

For examples -> For example
The autonomous system -> An autonomous
this sentence sounds a bit weird after enumerating examples of autonomous systems, i would put the definition of autonomous system before.
The autonomous navigation system needs -> An autonomous ... requires
It needs to know where am I -> where it is
where is safe -> where it is safe
"how to control" -> sounds weird to me, like a word is missing of something, i would say it should be rephrased
these autonomous technologies needs  -> ... need
statics, -> statistics ?
and sensing physics -> physics ? (i don't really understand "sensing physics")
After this sentence that presents why PythonRobotics is needed (autonomous systems requires lot of different technological background), it could a good idea to make a transition (with "in this paper, ...").
The unique point of this project has a lot of simulation animations that shows behaviors of each algorithm. -> you mean what makes it unique ? ("unique point" sounds weird to me)

I'm not sure that the github star history makes sense here.

~~~↑ DONE ↑ ~~~~

## Related works

state of the arts -> state of the art
beginners of robotics -> robotics beginners or beginners in robotics
or even simpler "it might be difficult for beginners of robotics" -> for beginners (not need to mention that this is robotics)
these papers doesn’t -> don't
because these papers doesn’t include implementation examples
such as sample codes. -> implementation examples. (if it is an implementation, I think it is clear that you are talking about code)
share it’s lecture notes -> its
These lecture’s -> These lectures'
lecture notes doesn’t include -> don't
ROS initially released from 2007 -> was ... in 2007
"However, it might has an issue, because of lacking of documentations about it’s basic algorithm." -> I understand the idea but i would rephrase it (also it's -> its, documentations -> documentation)
"Also, it might be written by a complicated programming language such as C++ for calculation performance and platform reasons. This is not good for focusing on learning robotics
algorithms." -> Those packages are usually written in C++ because the focus is computational efficiency and not readability, which makes learning from the code harder. (it is a proposition)

"It’s learning strategy is great for beginners to understand it’s technical background." -> its ... its

## Philosophy

"This project based on three main philosophies." -> It is based on three main ideas.

"that the codes" -> the code
"Therefore, the code have to be easy to read and understand the algorithm." -> repetition of the first sentence

"Programming language, Python" -> The Python programming language... because of its simplicity

"The second one is the algorithms which is practical and widely used in academia and industry are selected." -> is that implemented algorithms have to be practical and widely used in academia and industry.

"It allows us to try the codes" -> it allows to run code samples easily 
"as bellows" -> as bellow

## Repository structure

"the brief repository structure of this project is described." -> we briefly described the repository structure.
"has five directories which means five" -> corresponding to .. navigation: localization, mapping, SLAM, path planning and path tracking.
"Each directory includes several directories which has a sample code using different algorithms." -> There is one sub-directory per algorithm

described . -> described.

Localization is an ability -> is the ability
it's -> its position

is an ability -> the ability
it's -> its
imaging sensor -> camera ?
obstacle positions and it’ shape -> the position and shape of obstacles to avoid them.

In this paper, We introduced -> we introduced PythonRobotics, a code collection of robotics algorithms, with a focus on autonomous navigation.

**Comment by AtsushiSakai** [comment]
@araffin Oh. Great. Thank you for your feedback!! I appreciate.
I will fix it based on your feedback.

>Note: all the links (references) are not clickable, is that normal ?

Oh, you are right. Maybe because of tex environment. I'll check it.

**Comment by AtsushiSakai** [comment]
I fixed hyperlink issue.
